libraries/listen.rb

```diff
@@ -34,8 +34,8 @@ def self.converted_listen_ports_and_addresses(node)
       return [] unless node['apache']['listen_ports'] || node['apache']['listen_addresses']
       Chef::Log.warn "node['apache']['listen_ports'] and node['apache']['listen_addresses'] are deprecated in favor of node['apache']['listen']. Please adjust your cookbooks"
 
-      node['apache']['listen_addresses'].uniq.each_with_object([]) do |address, listen|
-        node['apache']['listen_ports'].uniq.each do |port|
+      (node['apache']['listen_addresses'] || []).uniq.each_with_object([]) do |address, listen|
+        (node['apache']['listen_ports'] || []).uniq.each do |port|
```

Comment:
I did not realize that they changed it to `||` in 3.2.2 (changeset 52372bbe74c25acea1021eb5f7d1d1df449c4fd6). In that case it probably needs some sane default for ports and addresses if either are nil. Maybe what it had in 3.1?

maybe they should be ```
      (node['apache']['listen_addresses'] || %w(*)).uniq.each_with_object([]) do |address, listen|
        (node['apache']['listen_ports'] || %w(80)).uniq.each do |port|
```